src/App.tsx

```diff
@@ -37,6 +37,104 @@ function App() {
   return (
     <Router>
       <AppHeader loggedIn={loggedIn} setUserId={setUserId} />
+      <link rel="preconnect" href="https://fonts.gstatic.com" />
+      <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&family=Pathway+Gothic+One&display=swap" rel="stylesheet" />
+      <Switch>
```

Comment:
This will cause problems. Don't just move things around without understanding how everything works.